discretisedfield/mesh.py

```diff
@@ -917,18 +917,32 @@ def __or__(self, other):
 
```

Comment:
Does point 1 mean `same discretisation cell size`?

Reply:
I do not fully understand what you mean?

Reply:
The explanation above (l. 916) says `... same discretisation cells` which could be interpreted as "both meshes have the same region and cell size -> the same cell**s**. We want to say that both meshes have "the same cell size". So maybe changing `cells` to `cell size` would be better?